apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx

```diff
@@ -131,14 +131,16 @@ export const EditMultiFactorAuthenticator: FunctionComponent<EditMultiFactorAuth
     const handleAuthenticatorConfigFormSubmit = (values: MultiFactorAuthenticatorInterface): void => {
         setIsSubmitting(true);
 
+        const i18nKeyForMFAuthenticator: string = getI18nKeyForMultiFactorAuthenticator(authenticator);
```

Comment:
```suggestion
        const i18nKeyForMFAAuthenticator: string = getI18nKeyForMultiFactorAuthenticator(authenticator);
```

Reply:
The use of MFAuthenticator is intentional here "A" in MFAAuthenticator would also mean authentication. But I agree it's better to stick with the conventional shorthand form.

Reply:
Resolved in ea21716da10d86b7eb72ee5acfbe706711021e98